### PR TITLE
Specify the MP_RTT Age parameter

### PR DESCRIPTION
**Pull request by markusa**

### CODE REVIEW

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 1]
```diff
@@ -748,14 +751,13 @@ Smooth RTT (=3)
...
-Age (=4)
-: The Age parameter is a 4-byte value which is set to the age or  
-  timestamp when the Ack for the datagram was received in case of  
-  RTT type = 0 and may contain the periods for computing of derived  
-  RTT values depending on other RTT types, i.e., the Minimum (=1)  
-  and Maximum (=2) as well as the averaged smoothed RTT value (=3).  
+Age
+: The Age parameter is a 4-byte value which is set to the age and  
+  contains the periods for computing of derived  
+  RTT values for the Minimum (=1), Maximum (=2) and
```
Comment (abrunstrom) [inline draft-ietf-tsvwg-multipath-dccp.mkd:757]:
```suggestion
  RTT values for the Minimum (=1), Maximum (=2)
```

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 2]
```diff
@@ -748,14 +751,13 @@ Smooth RTT (=3)
...
 {: vspace='0'}
 
-Age (=4)
-: The Age parameter is a 4-byte value which is set to the age or  
-  timestamp when the Ack for the datagram was received in case of  
-  RTT type = 0 and may contain the periods for computing of derived  
-  RTT values depending on other RTT types, i.e., the Minimum (=1)  
-  and Maximum (=2) as well as the averaged smoothed RTT value (=3).  
+Age
+: The Age parameter is a 4-byte value which is set to the age and
```
Comment (abrunstrom) [inline draft-ietf-tsvwg-multipath-dccp.mkd:755]: It was already said above that the Age information is a 32-bit integer so the 4-byte here is now redundant.

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 3]
```diff
@@ -724,10 +724,13 @@ The MP_RTT option is used to transmit RTT values in milliseconds and
 MUST belong to the path over which this information is transmitted.
 Additionally, the age of the measurement is specified in milliseconds.
 
+The RTT and Age information is a 32-bit integer that increases monotonically with
+time, at a rate of 1 unit per 1 millisecond.  At this rate,
+RTT/Age Value will cover approximately 1193 hours.
+
```
Comment (abrunstrom) [inline draft-ietf-tsvwg-multipath-dccp.mkd:730]: This reads strangely to me. What are we trying to say here? That the Age and RTT are represented as 32-bit integers? And that this allows to capture a maximum value for the RTT and Age of approximately 1193 hours. We already say above that they use milliseconds as the unit, so I do not understand the part about the rate.

**Review by abrunstrom — CHANGES_REQUESTED** [review]
Some edits needed.

File: draft-ietf-tsvwg-multipath-dccp.mkd (hunk 2 above)
Comment (markusa) [reply draft-ietf-tsvwg-multipath-dccp.mkd]: Please look into https://github.com/markusa/ietf-multipath-dccp/pull/52/commits/ece3530163cf2b46fd0f8c09264a17a03b21782d

File: draft-ietf-tsvwg-multipath-dccp.mkd (hunk 3 above)
Comment (markusa) [reply draft-ietf-tsvwg-multipath-dccp.mkd:729]: Please look into https://github.com/markusa/ietf-multipath-dccp/pull/52/commits/fca488a5d37a2e5d3e67745b5a136b608fb73998

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 4]
```diff
@@ -724,10 +724,12 @@ The MP_RTT option is used to transmit RTT values in milliseconds and
 MUST belong to the path over which this information is transmitted.
 Additionally, the age of the measurement is specified in milliseconds.
 
+The RTT and Age information is a 32-bit integer used to cover a period of
```
Comment (abrunstrom) [inline draft-ietf-tsvwg-multipath-dccp.mkd:727]:
```suggestion
The RTT and Age information is a 32-bit integer, which allows to cover a period of
```

**Review by abrunstrom — APPROVED** [review]
Suggested a small edit, otherwise looks fine.

**Pull request merged**